website/content/en/preview/concepts/disruption.md

```diff
@@ -177,6 +181,63 @@ To enable interruption handling, configure the `--interruption-queue-name` CLI a
 
 ## Controls
 
+### Disruption Budgets
+
+You can rate limit Karpenter's disruption through the `NodePool.Spec.Disruption.Budgets`. If undefined, Karpenter will default to one budget with `nodes: 10%`. Budgets will take into account nodes that are being deleted for any reason, and will only block Karpenter from disrupting nodes voluntarily through expiration, drift, emptiness, and consolidation.
+
+#### budget.Duration + budget.Schedule
+Duration and Schedule can only be defined together. When omitted, the budget is always active. When defined, the schedule determines when a budget is active, and the duration determines how long from that point the budget is active. 
+
+##### Schedule
+Schedule is a cronjob schedule. Generally, the cron syntax is five space-delimited values with options below. 
+Follow the [Kubernetes documentation](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#writing-a-cronjob-spec) for how to follow the cron syntax. 
+
+```bash
+# ┌───────────── minute (0 - 59)
+# │ ┌───────────── hour (0 - 23)
+# │ │ ┌───────────── day of the month (1 - 31)
+# │ │ │ ┌───────────── month (1 - 12)
+# │ │ │ │ ┌───────────── day of the week (0 - 6) (Sunday to Saturday;
+# │ │ │ │ │                                   7 is also Sunday on some systems)
+# │ │ │ │ │                                   OR sun, mon, tue, wed, thu, fri, sat
+# │ │ │ │ │
+# * * * * *
+```
+
+{{% alert title="Note" color="primary" %}}
+Timezones are not supported. Most images default to UTC, but it is best practice to ensure this is the case when considering how to define your budgets. 
```

Comment:
This statement is confusing to me. Can we not guarantee that we are _always_ using UTC when we are doing the cron syntax? If this isn't always the case and we aren't using the same time format across images, we're going to get customer issues for disrupting when we shouldn't be

Reply:
From what I can tell, it isn't guaranteed that all images are using UTC. Anywhere I could find, it all pointed to using the default UTC, but I want to have a note here for anyone specifying timezones within their images. Thoughts?

Reply:
> I could imagine users who are specifying timezone in their image be very surprised if Karpenter logs in that timezone, but then uses UTC for calculations

I don't think this is surprising to anyone, really. I think a better question is probably how upstream handles this for the cronjob-controller. My expectation was that this time was always reasoned in terms of UTC

Reply:
Discussed offline. The container filesystem itself should contain information on which timezone that Karpenter is running in. By default, this timezone should be set as UTC but this overridable through a volume mount. It makes sense to me that we would just adhere to the container timezone and stick with UTC for the vast majority of use-cases: https://stackoverflow.com/questions/57607381/how-do-i-change-timezone-in-a-docker-container